app/src/main/java/tech/pauly/findapet/data/models/Sex.java

```diff
@@ -2,5 +2,6 @@
 
 public enum Sex {
     M,
-    F
+    F,
+    U
```

Comment:
Don't ask me what this is...apparently there's asexual birds flying around in Michigan.